addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -833,9 +871,33 @@ public void run() {
             if (getThing().getStatus() == ThingStatus.ONLINE) {
                 if (isAwake()) {
                     requestData(TESLA_CHARGE_STATE);
+                    try {
+                        Thread.sleep(1000);
```

Comment:
@kaikreuzer unfortunately it is here to stay because the Tesla back-end does not respect the order and timing of requests being made. I suspect this to because of load balancers at their end. Various calls to their back-end return the same kind of data. They use timestamps in their responses, and I do track these in the binding, ignoring data that is arriving out of sequence. So, in order to make sure that the responses come back in order I had to include this Thread.sleep(1000).